src/components/UI/FileDrop/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { FileDropProps } from "@/types";
+import s from "./FileDrop.module.scss"
+import { useRef, useState } from "react";
+
+export const FileDrop = ({
+  children,
+  className,
+  onChange = (items) => { console.log('Changed Items: ', items) },
```

Comment:
Why did you leave the console log here?

Reply:
so that the prop has a default function. Im sure its not good to leave it empty or make it compolsory so should i just pass a '()=>{}' as default or is it better to leave a console.log?

Reply:
Better empty function.